docs/collections/documents/document-store-example.md

```diff
@@ -578,89 +739,139 @@ if __name__ == '__main__':
+
+function messageHandler (error) {
+  const message = {
+    "StatusCode ": error.statusCode,
+    "ErrorMessage ": error.message,
+    "ErrorNum ": error.errorNum,
+  };
+  console.log(message);
+}
+
+//Variables
+const collectionName = "employees";
+
+//Queries
+const insertData =
+  "INSERT {'firstname':@firstname, 'lastname':@lastname, 'email':@email, 'zipcode':@zipcode, '_key': 'abc'} IN " +
+  collectionName;
+
+const getData = "FOR doc IN " + collectionName + " RETURN doc";
+
+const updateData =
+  "UPDATE 'abc' WITH {'lastname': @lastname } IN " + collectionName;
+
+const deleteData = "REMOVE 'abc' IN " + collectionName;
+
+const getCount = "RETURN COUNT(FOR doc IN " + collectionName + " RETURN 1)";
+
+async function restqldemo () {
+  await client
+    .login(email, password)
+    .then(() => console.log("\n1. User authentication done!"))
+    .catch((error) => messageHandler(error));
+
+  console.log("\n------- SAVING THE QUERIES  ------");
+
+  await client
+    .createRestql("insertData", insertData, {})
+    .then((restQl) => {
+      console.log("2. " + restQl.result.name + "query created.");
+    })
+    .catch((error) => messageHandler(error));
+  await client
+    .createRestql("getData", getData, {})
+    .then((restQl) =>
+      console.log("3. " + restQl.result.name + "query created.")
+    )
+    .catch((error) => messageHandler(error));
+  await client
+    .createRestql("updateData", updateData, {})
+    .then((restQl) =>
+      console.log("4. " + restQl.result.name + "query created.")
+    )
+    .catch((error) => messageHandler(error));
+  await client
+    .createRestql("deleteData", deleteData, {})
+    .then((restQl) =>
+      console.log("5. " + restQl.result.name + "query created.")
+    )
+    .catch((error) => messageHandler(error));
+  await client
+    .createRestql("getCount", getCount, {})
+    .then((restQl) =>
+      console.log("6. " + restQl.result.name + "query created.")
+    )
+    .catch((error) => messageHandler(error));
+
+  console.log("\n------- EXECUTING THE QUERIES  ------");
+
+  const bindVars = {
+    firstname: "john",
+    lastname: "doe",
+    email: "john.doe@macrometa.io",
+    zipcode: "511037",
+  };
+
+  await client
+    .executeRestql("insertData", bindVars)
+    .then(() => console.log("\n7. Data inserted."))
+    .catch((error) => messageHandler(error));
+
+  await client
+    .executeRestql("getData")
+    .then((res) => {
+      console.log("\n8. getData query result: ");
+      console.log(res.result);
+    })
+    .catch((error) => messageHandler(error));
+
+  await client
+    .executeRestql("updateData", { lastname: "mathews" })
+    .then(() => console.log("\n9. Data updated."))
+    .catch((error) => messageHandler(error));
+
+  await client
+    .executeRestql("getData")
+    .then((res) => {
+      console.log("\n10. Output of get data query after update: ");
```

Comment:
```suggestion
      console.log("\n10. Get output after update: ");
```

@diana-macrometa This one can probably be improved, any thoughts?

Reply:
That's a toughie. The type of query is getData, so maybe use the proper name in the log to make clearer what is happening? If I'm reading this right, this sentence is right before the posting of the output of said query. Maybe:

Updated getData query output:

Reply:
Good idea. Thanks!